### PR TITLE
Update breakdown to use icons (and other fun updates)

### LINKED ISSUE #20: Update hatch panel and cargo rows to use icons
Icons from: https://material.io/tools/icons/?style=baseline

Null hatch panel: `trip_origin`/`#616161`
Hatch panel: `trip_origin`/`#ffeb3b`
Cargo: `lens`/`#ff6d00`

### PR DESCRIPTION
**Pull request by dracco1993**
## Description
Update the 2019 breakdown to be cool, like the other ones.  😅 

## Motivation and Context
The iOS breakdowns were added in a simple state, but then the web and Android ones made cool improvements, that our iOS users deserve to have as well.
Closes #20

## How Has This Been Tested?
Mostly just locally. (see below)

## Screenshots
simulator screen shot - iphone xr - 2019-03-07 at 18 37 54
simulator screen shot - iphone xr - 2019-03-07 at 18 41 24

### CODE REVIEW

File: app/views/breakdowns/MatchBreakdown.js [hunk 1]
```diff
@@ -36,4 +36,22 @@ export default class MatchBreakdown extends React.Component {
       <Image source={images.arrows.down} />
     );
   }
+
+  nullHatchPanelImage() {
```
Comment (ZachOrr) [inline app/views/breakdowns/MatchBreakdown.js]: Can probably be in the 2019-specific match breakdown, yeah?

Comment (dracco1993) [reply app/views/breakdowns/MatchBreakdown.js]: Good point, I'll get that moved.

File: app/components/ImageCount.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {
+  Text,
+  View
+} from 'react-native';
+
+export default class InsightRow extends React.Component {
```
Comment (ZachOrr) [inline app/components/ImageCount.js]: This name seems wrong

Comment (dracco1993) [reply app/components/ImageCount.js]: 🤦‍♂️

File: app/views/breakdowns/MatchBreakdown2019.js [hunk 3]
```diff
@@ -1,44 +1,100 @@
...
+  Text
 } from 'react-native';
 import BreakdownRow from '../../components/BreakdownRow';
+import images from '../../config/images';
+import ImageCount from '../../components/ImageCount';
 import MatchBreakdown from '../breakdowns/MatchBreakdown';
 import breakdownStyle from '../../styles/breakdown';
 
+// Override our Image and Text to have specific sizes
+const Image = ({ style, ...props }) => <ReactNative.Image style={[breakdownStyle.imageSize, style]} {...props} />;
```
Comment (ZachOrr) [inline app/views/breakdowns/MatchBreakdown2019.js:14]: You don't need to do this here but - for the love of god let's DRY this

Comment (dracco1993) [reply app/views/breakdowns/MatchBreakdown2019.js:14]: We do, here:
https://github.com/the-blue-alliance/the-blue-alliance-react/pull/28/files#diff-c050433af16119a1bae81c576f1d18ddR20

**Pull request merged**